tools/enumerate-devices/rs-enumerate-devices.cpp

```diff
@@ -81,6 +86,15 @@ void safe_get_intrinsics(const video_stream_profile& profile, rs2_intrinsics& in
     {}
 }
 
+void safe_get_motion_intrinsics(const motion_stream_profile& profile, rs2_motion_device_intrinsic& intrinsics)
```

Comment:
There is no way to know if the function fails. A Boolean return value will do the work.
Also, please consider to change the function name to "try_get...()"